app/models/media.rb

```diff
@@ -75,6 +75,7 @@ def as_json(options = {})
     if options.delete(:force) || Pender::Store.current.read(Media.get_id(self.original_url), :json).nil?
       handle_exceptions(self, StandardError) { self.parse }
       self.data['title'] = self.url if self.data['title'].blank?
+      self.data['description'] = self.url if self.data['description'].blank?
```

Comment:
Since it seems like we've been asking for the description to be set to the URL by default as well as title, I went ahead and made the change here, which will impact all parsers.

Reply:
Makes sense, thanks!